### ISSUE TITLE
{Linked:Bug9528434} The interactive control in the @workspace tooltip is not accessible using keyboard (Tab or Arrow) keys: A11y_VS Code_Copilot Chat_Keyboard

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#237177]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-VisualStudioCodeClient-Win32-Jan2024; #Visual Studio Code Client; #Win32; #Win11; #WCAG2.1.1; #FTP; #Keyboard; #ED:Bug9528434

### Environment and OS details:
Version: 1.97.0-insider (user setup)
Windows 11 Enterprise Version 24H2
OS Build: 26100.2605

### Repro Steps:

1. Open Visual Studio Code Insiders editor.
2. Tab till Copilot chat Window and press enter.
3. Navigate to Chat(Ctrl+Alt+I) in Left Nav and activate it.
4. Navigate to Ask copilot text field type /Help Command and press enter.
5. Navigate to @workspace Bulleted List.
6. Observed that the interactive control in the @workspace tooltip is not accessible using keyboard.

### Actual Result:
 The interactive control in the @workspace tooltip is not accessible using keyboard (Tab or Arrow) keys.

### Expected Result:
 The interactive control in the @workspace tooltip should be accessible with keyboard (Tab or Arrow) keys.

### User Impact:
Keyboard users will face difficulty if the interactive control in the @workspace tooltip is not accessible using keyboard.

### Attachment:
![Image](https://github.com/user-attachments/assets/e6f99d83-71b0-4cc0-b965-962183c2c337)

**Comment by meganrogge** [comment]
Now, that `@workspace` is not hoverable. 

The header item is focusable with the keyboard. I suggest closing this as it's fixed.
https://github.com/user-attachments/assets/fc701157-3a83-4642-a891-ce4b327052f3

**Comment by kupatkar99** [comment]
verified the issue in below environment:
version 1.97.0(user setup)
issue is still repro'ing. Please find the below added attachment for reference.
Could you please confirm us the fixes are available in which version or Do we have any shortcut key to access the controls inside the @Workspace tooltip? Thanks!

![Image](https://github.com/user-attachments/assets/8bc22a5d-1467-4e9e-9d36-9bc044fb8e58)

**Comment by benibenj** [comment]
The next insiders release will support the action `workbench.action.showHower` (`ctrl+k | ctrl+i`) which allows to show and focus the hover for the focused element.

**Comment by kjawale99** [comment]
@meganrogge This bug is about the @workspace that appears in blue color and not for the header. Can you please check again.
